ui/apps/ui/src/app/collections/data/datasets/adapter.data.ts

```diff
@@ -25,6 +26,9 @@ export const datasetsAdapter: IAdapter = {
     id: openAIREResult.id,
     title: openAIREResult?.title?.join(' ') || '',
     description: openAIREResult?.description?.join(' ') || '',
+    date: openAIREResult['publication_date']
+      ? moment(openAIREResult['publication_date']).format('DD MMMM YYYY')
+      : '',
```

Comment:
This piece of code is repeated multiple times - it is a prime target for creating a function out of it.

Additionally I would suggest creating `export const DATE_FORMAT = 'DD MMMM YYYY'` and using it instead of repeating string `DD MMMM YYYY` every time. If at some point in the future we do decide to change date formatting we'll have to hunt down all those string literals.